### ISSUE TITLE
Case insensitive search for  Bibtex cite keys

### ISSUE THREAD

**Issue by hofingermarkus** [texstudio-org/texstudio#2303]
### Describe the feature and the current behavior/state
Currently when I use the automatic completion of TexStudio to complete citation keys it appears to require exact case matching.
This can lead to the impression, that either an entry isn't there or TexStudio does not find it, if there is a case mismatch.
It would be great if the search were case insensitive instead, i.e. it should find a matching key no matter if it is upper or lower case.

### Who will benefit with this feature?
Basically everyone using the citation search - because not finding a citation key because of a case mismatch is not intuitive to the user and causes confusion on whey the key does not show up.
Alternatively provide a warning text on the popup to inform that this is a case  sensitive search!

### Any Other info
The following feature enhancement reported that the search is case sensitive
https://github.com/texstudio-org/texstudio/issues/132

TexStudio 4.2.2
Qt 5.9.5
Ubuntu

**Comment by peterChri** [comment]
I'd appreciate this feature very much, its tiresome doing the search for cite keys in a case sensitive manner.
